packages/e2e-tests/src/specs/dashboard/telemetryBanner.js

```diff
@@ -69,7 +69,11 @@ describe('Telemetry Banner', () => {
 
   it('should keep focus on the checkbox when checking/unchecking via keyboard', async () => {
     await page.waitForTimeout(300); // Because dashboard takes some time before banner appears.
-    const checkbox = await page.$('#telemetry-banner-opt-in');
+
+    const bannerSelector = '#telemetry-banner-opt-in';
+    await page.waitForSelector(bannerSelector);
+    await expect(page).toMatchElement(bannerSelector);
+    const checkbox = await page.$(bannerSelector);
```

Comment:
Is the `waitForTimeout` above still needed now?

Is `waitForSelector` really needed? `toMatchElement` does already wait for the selector, and the timeout for it can be customized if desired.

Reply:
It was me testing a workaround for #9619. it is removed now.